package.json

```diff
@@ -53,7 +53,9 @@
     "webpack": "^1.7.3"
   },
   "dependencies": {
-    "react": "0.13.1",
+    "classnames": "^1.2.0",
+    "hammerjs": "^2.0.4",
+    "react": "^0.13.1",
```

Comment:
I usually like to version lock non dev-deps because I don't trust people to not introduce breaking changes. :dancers: